src/lib/bigNums.js

```diff
@@ -1,5 +1,5 @@
 "use strict";
-const Big = require("big.js");
+const Big = require("../lib/fixedDecimal.js");
```

Comment:
housekeeping for future readability:  "Big" variable should be renamed to FixedDecimal (maybe the class name should be FixedDecimal too, capital indicating it's a class)